template/core/commons/src/main/java/io/bloco/core/commons/ResultUtils.kt

```diff
@@ -0,0 +1,38 @@
+package io.bloco.core.commons
```

Comment:
I get that these Utils are useful, but could we do without them in the Template? It's not a sure thing that we will use the Kotlin Result in all our projects...